core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -1156,6 +1158,433 @@ describe('Contributions and review component', () => {
+                              }
+                            },
+                            rows: { value: 1 },
+                            catchMisspellings: {
+                              value: false
+                            }
+                          },
+                          default_outcome: {
+                            dest: null,
+                            dest_if_really_stuck: null,
+                            feedback: {
+                              html: 'Correct Answer',
+                              content_id: 'content_2'
+                            },
+                            param_changes: [],
+                            labelled_as_correct: true
+                          },
+                          hints: [{
+                            hint_content: {
+                              html: 'Hint 1',
+                              content_id: 'content_3'
+                            }
+                          }],
+                          solution: {
+                            correct_answer: 'This is the correct answer',
+                            answer_is_exclusive: false,
+                            explanation: {
+                              html: 'Solution explanation',
+                              content_id: 'content_4'
+                            }
+                          },
+                          id: 'TextInput'
+                        },
+                        param_changes: [],
+                        recorded_voiceovers: {
+                          voiceovers_mapping: {}
+                        }
+                      },
+                    },
+                    content_html: 'Translation',
+                    translation_html: 'Tradução',
+                    skill_id: 'skill_id'
+                  },
+                  status: 'rejected',
+                  exploration_content_html: null
+                },
+                details: {
+                  topic_name: 'topic_name',
+                  story_title: 'story_title',
+                  chapter_title: 'chapter_title'
+                }
+              }
+            },
+            more: false
+          }));
+
+        component.switchToTab(
+          component.TAB_TYPE_REVIEWS, 'translate_content');
+        // Set up contributions with a translation.
+        component.loadContributions(null).then(({opportunitiesDicts, more}) => {
+          expect(Object.keys(component.contributions)).toContain(
+            'suggestion_1');
+          expect(opportunitiesDicts).toEqual([{
+            id: 'suggestion_1',
+            heading: 'Tradução',
+            subheading: 'topic_name / story_title / chapter_title',
+            labelText: 'Obsolete',
+            labelColor: '#e76c8c',
+            actionButtonTitle: 'Review',
+            translationWordCount: undefined
+          }]);
+          expect(more).toEqual(false);
+          // When opening the review modal for translations,
```

Comment:
Nit: Add new line above to help differentiate this different sub-test. Ditto below.

Reply:
Done